test.json

```diff
@@ -0,0 +1,3 @@
+{
+  "text": "Styleguidesdsdsd"
```

Comment:
Needs to be an actual config.json file with the actual config variables? Why did you undo it?